src/main.ts

```diff
@@ -56,18 +57,24 @@ import {
   GOOGLE_SSH_KEYS_TEMP_DIR_VAR,
 } from './const';
 
-export const GCLOUD_METRICS_ENV_VAR = 'CLOUDSDK_METRICS_ENVIRONMENT';
-export const GCLOUD_METRICS_LABEL = 'github-actions-ssh-compute';
+// Do not listen to the linter - this can NOT be rewritten as an ES6 import
+// statement.
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version: appVersion } = require('../package.json');
 
 export async function run(): Promise<void> {
-  try {
-    exportVariable(GCLOUD_METRICS_ENV_VAR, GCLOUD_METRICS_LABEL);
-
-    // Warn if pinned to HEAD
-    if (isPinnedToHead()) {
-      logWarning(pinnedToHeadWarning('v0'));
-    }
+  // Register metrics
+  const restoreEnv = stubEnv({
+    CLOUDSDK_METRICS_ENVIRONMENT: 'github-actions-ssh-compute',
+    CLOUDSDK_METRICS_ENVIRONMENT_VERSION: appVersion,
+  });
+
+  // Warn if pinned to HEAD
+  if (isPinnedToHead()) {
+    logWarning(pinnedToHeadWarning('v0'));
```

Comment:
Should this be v1?

Reply:
No, there's no v1 cut for this repo, it's still on the v0 series.